gradle.properties

```diff
@@ -3,7 +3,7 @@ liquibaseTaskPrefix=liquibase
 # The following properties control the environment for building Docker images. To point tasks to Minikube's Docker Daemon,
 # run `docker port minikube` and look for mapping of 2376; run `minikube docker-env` and look for `$Env:DOCKER_CERT_PATH`
 #build.docker.host=https://127.0.0.1:49156
-#build.docker.tls-verify=true
+#build.docker.tsl-verify=true
```

Comment:
Seems to have been rebased incrorectly

Reply:
I changed it manually, I remember that there was a typo. Copied it from `README.md`
Will revert and fix in `README.md`

Reply:
Oh, I saw Gradle wrapper being downgraded and thought it was a bad rebase. Is Gradle downgraded because of that error with Kotlin plugin?

Reply:
yeap, I revert to 7.5.1
Will extract this change to a dedicated PR

Reply:
#1604